src/open_inwoner/openklant/tests/test_views.py

```diff
@@ -57,7 +58,7 @@ def test_list_for_bsn(self, m, mock_get_kcm_answer_mapping):
 
         detail_url = reverse(
             "cases:contactmoment_detail",
-            kwargs={"kcm_uuid": data.klant_contactmoment["uuid"]},
+            kwargs={"kcm_uuid": uuid_from_url(data.klant_contactmoment["uuid"])},
```

Comment:
I think you forgot to remove the `uuid` from the `klant_contactmoment` (applies to other cases in this test file as well)